leo/core/leoAtFile.py

```diff
@@ -2540,27 +2596,25 @@ def precheck(self, fileName: str, root: Position) -> bool:  # pragma: no cover
         Return True if so.  Return False *and* issue a warning otherwise.
         """
         at = self
-        #
+        if g.unitTesting:
+            return True
         # #1450: First, check that the directory exists.
         theDir = g.os_path_dirname(fileName)
         if theDir and not g.os_path_exists(theDir):
             at.error(f"Directory not found:\n{theDir}")
             return False
-        #
         # Now check the file.
         if not at.shouldPromptForDangerousWrite(fileName, root):
             # Fix bug 889175: Remember the full fileName.
```

Comment:
Should this 'bug fix' be replaced with a single '#' as was done in other instances of this commit for this file?

Reply:
My intended standard is `# #<issue-number>`.